src/System/PairExchange.jl

```diff
@@ -133,36 +93,15 @@ function set_exchange!(sys::System{N}, J, bond::Bond) where N
     end
 
     # Print a warning if an interaction already exists for bond
-    if any(x -> x.bond == bond, vcat(ints[bond.i].heisen, ints[bond.i].exchange))
+    if any(x -> x.bond == bond, ints[bond.i].pair)
```

Comment:
Feature request; flag to disable this warning. I use this to re-run the forwards problem with different `J` each time and currently no way to turn this off easily

Reply:
Good idea. Per Slack discussion, let's address this in a follow-up PR using https://docs.julialang.org/en/v1/stdlib/Logging/